Tests/ValetIntegrationTests/SinglePromptSecureEnclaveIntegrationTests.swift

```diff
@@ -160,6 +160,8 @@ class SinglePromptSecureEnclaveIntegrationTests: XCTestCase
         }
     }
 
+    #if !os(macOS)
```

Comment:
Can we use an `@available` here instead?

Reply:
We can't use `@available` or `#available` to exclude a platform. Both require a trailing `*` which opts into all platforms that are not mentioned.